irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,236 @@
+$('#btn-signin-irods').on('click',function(){
+    if(sessionStorage.username)
+        $("#username").val(sessionStorage.username);
+    if(sessionStorage.password)
+        $("#password").val(sessionStorage.password);
+    if(sessionStorage.host)
+        $("#host").val(sessionStorage.host);
+    if(sessionStorage.port)
+        $("#port").val(sessionStorage.port);
+    if(sessionStorage.zone)
+        $("#zone").val(sessionStorage.zone);
+});
+
+$('#btn-select-irods-file').on('click',function(){
+    $('#res_type').val($('#resource-type').val());
+    $('#file_struct').children().remove();
+    if (sessionStorage.signininfo) {
+        $("#irods_content_label").text(sessionStorage.username);
+        $('#root_store').val(sessionStorage.datastore);
+    }
+	store = $('#root_store').val();
+	// Setting up the view tab
+	$('#file_struct').attr('name',store);
+	$('#irods_view_store').val(store);
+	// loading file structs
+	parent = $('#file_struct');
```

Comment:
if you do this at the top of the function you can use the variable instead of querying the same selector for the element 3 separate times.